smoketest.bash

```diff
@@ -181,7 +215,18 @@ cleanup() {
         down "${downFlags[@]}"
     if [ "${USE_PROXY}" = "true" ]; then
         ${container_engine} rm proxy_cfg_helper || true
+        ${container_engine} rm proxy_certs_helper || true
         ${container_engine} volume rm auth_proxy_cfg || true
+        ${container_engine} volume rm auth_proxy_certs || true
+    fi
+    if [ "${INCLUDE_SAMPLE_APPS}" = "true" ] && [ "${SAMPLE_APPS_USE_TLS}" = "true" ]; then
+        rm ${DIR}/compose/agent_certs/agent_server.cer
+        rm ${DIR}/compose/agent_certs/agent-keystore.p12
+        rm ${DIR}/compose/agent_certs/keystore.pass
+    fi
+    if [ "${USE_TLS}" = "true" ]; then
+        rm ${DIR}/compose/auth_certs/certificate.pem
```

Comment:
These instances of `${DIR}` should be quoted like `"${DIR}"` for safety. If the `$DIR` value happens to contain a ` ` space character, then it'd break this line in potentially dangerous ways, since the `rm` invocation would end up getting called with two separate arguments.